recipe/meta.yaml

```
@@ -39,9 +39,9 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.40.*
+    - llvmlite 0.41.*
     - numpy
-    - tbb-devel 2021.6.0
+    - tbb-devel >=2021.6.0
```

Let's keep this as-is. We already use `pin_compatible` below to ensure users can install any version later than this

```suggestion
    - tbb-devel 2021.6.0
```